src/repository/views.py

```diff
@@ -879,9 +880,11 @@ def repository_manager_article(request, preprint_id):
                     'You must assign at least one galley file.',
                 )
             else:
+                d = datetime.fromisoformat(request.POST.get('datetime', timezone.now().strftime("%Y-%m-%d %H:%M")))
```

Comment:
What happens if a malformed date is passed in here?

Reply:
@ajrbyers this scenario is unlikely because the form field does inline checking but I added a catch to handle this situation.